src/Service.Tests/CosmosTests/QueryFilterTests.cs

```diff
@@ -647,6 +649,28 @@ public async Task TestFilterOnInnerNestedFields()
             await ExecuteAndValidateResult(_graphQLQueryName, gqlQuery, dbQuery);
         }
 
+        /// <summary>
+        /// Test filters when entity name is using alias
```

Comment:
Can you note what is the alias versus non-alias? 

Without cross-referencing the config in a separate file, it isn't immediately obvious what value represents the alias you are validating. In this test, it seems that the usage of `tag` as a (nested?) entity name in the filter validates that the value `tag` is recognized as the GraphQL object type name for the entity `TagAlias`